book/graphics.md

```diff
@@ -309,34 +310,36 @@ Here's a widget demonstrating that concept:
 :::
 
 ::: {.print-only}
-![A flow-chart of how characters the cursor moves as each character is drawn](im/graphics-cursor.png)
+![Figure 3: A flow-chart of how the cursor moves as each character is drawn.](im/graphics-cursor.png)
 :::
 
 [^crlf]: In the olden days of typewriters, increasing *y* meant
     *feed*ing in a new *line*, and resetting *x* meant *return*ing the
     *carriage* that printed letters to the left edge of the page. So
-    ASCII standardizes two separate characters—"carriage return" and
+    the American Standard Code for Information Interchange ([ASCII][ascii]) standardized two separate characters—"carriage return" and
     "line feed"—for these operations, so that ASCII could be directly
     executed by teletypewriters. That's why headers in HTTP are
     separated by `\r\n`, even though modern computers have no
     mechanical carriage.
 
+[ascii]: https://en.wikipedia.org/wiki/ASCII
+
 [^not-800]: Not 800, because we started at pixel 13 and I want to leave an
     even gap on both sides.
 
-At this point you should be able to load up [this page][ex-monkey] in your
-browser and have it look about like this:
+At this point you should be able to load up [our example ][ex-monkey] in your
+browser and have it look something like Figure 4.
 
 <div class=center>
-![Screenshot of Chinese text of 'Journey to the West'](examples/example2-text-screenshot.png)
+![Figure 4: The first chapter of *Journey to the West* rendered in our browser.](examples/example2-text-screenshot.png)
```

Comment:
The copyeditor said "Example page" instead of "The first chapter of *Journey to the West*" and said "in our canvas" instead of "in our browser". I think this is better, but willing to be convinced otherwise.